table.scss

```diff
@@ -0,0 +1,126 @@
+@import 'bower_components/vui-colors/colors.scss';
+@import 'bower_components/vui-typography/typography.scss';
+@import 'bower_components/vui-typography/px-to-base-rem.scss';
+
+@mixin vui-table() {
+	background-color: #ffffff;
+	border: 1px solid $vui-color-titanius;
+	border-collapse: separate;
+	border-radius: 6px;
```

Comment:
I think so - if the base is smaller/larger, things like padding and radius would change accordingly so that proportions remain the same.  We've done that elsewhere.  Feel free to double check with him.